src/BiometryService.Uno.Shared/BiometryExceptionReason.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace BiometryService;
+
+/// <summary>
+/// This enum helps categorize all possibles reasons that <see cref="BiometryException"/> has been raised.
+/// </summary>
+public enum BiometryExceptionReason
```

Comment:
I think each value should have text explaining how to fix each problem.

Reply:
I feel like `Unavailable`, `NotEnrolled`, `PasscodeNeeded` summaries are pretty self explanatory. Probably don't need to say more.  And `Failed` is for multiple errors.